server/tests/route.test.ts

```diff
@@ -166,6 +166,27 @@ describe('Get/logout', () => {
     expect(response.body.message).toBe('logged out successfully!');
   });
 });
+describe('Get/campaign/:id', () => {
+  test('campaign/:id', async () => {
+    const id = 1;
+    const data = {
+      id: 1,
+      title: 'Helping poor families',
+      description: 'This campaign helps save an amount of money that guarantees 50 families for two months',
+      target: 50000,
+      is_available: true,
+    };
+    const response = await request(app)
+      .get(`/api/campaign/${id}`)
+      .expect(200);
+    expect(response.body.data.id).toBe(data.id);
+    expect(response.body.data.title).toBe(data.title);
+    expect(response.body.data.description).toBe(data.description);
```

Comment:
Why not test them in just one line, I mean you have `response.body` and it should equal what you want.
And test the other cases.